src/appleseed.cli/main.cpp

```diff
@@ -264,7 +264,7 @@ namespace
         {
             params.insert_path(
                 "uniform_pixel_renderer.samples",
-                g_cl.m_samples.values()[1]);
+                g_cl.m_samples.values());
```

Comment:
That doesn't look right. You're inserting a `std::vector` as a value to the `"uniform_pixel_renderer.samples"` param. You probably need to use `value()` instead (to be checked).

How did you test your changes?

Reply:
The test suite is in `sandbox/tests/test suite/` but it wouldn't cover that. Manually running `appleseed.cli` and checking the render and/or the log should be enough. Can't hurt adding a couple `printf()`'s if needed.

I think this works because we actually _do_ support automatic conversion of `std::vector` to string. Still, it would be nicer to use `value()` (which returns a scalar) instead of `values()` (which in this case returns a `std::vector` of size 1).